tests/functional/src/main/java/org/mule/functional/classloading/isolation/maven/FileDependencyGraphProvider.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.functional.classloading.isolation.maven;
+
+import static java.nio.charset.Charset.defaultCharset;
+import static java.nio.file.Files.getFileAttributeView;
+import static java.nio.file.Files.readAllLines;
+import static java.nio.file.Paths.get;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.attribute.BasicFileAttributeView;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Reads the {@link FileDependencyGraphProvider#DEPENDENCIES_GRAPH_FILE_NAME} generated by the
```

Comment:
I think {@link FileDependencyGraphProvider#DEPENDENCIES_GRAPH_FILE_NAME} should be {@value FileDependencyGraphProvider#DEPENDENCIES_GRAPH_FILE_NAME}